### PR TITLE
Model integration

### PR DESCRIPTION
**Pull request by apurvnagvenkar**

### CODE REVIEW

File: models/crf/output_generation/city.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    list_json = []
+    read_line = 0
+    while read_line < (len(crf_data)):
+        if crf_data[read_line][1] == model_constant.CITY_O:
+            pass
+        else:
+            temp_list = check_label(reader_list=crf_data, reader_pointer=read_line,
+                                    begin_label=model_constant.CITY_FROM_B, inner_label=model_constant.CITY_FROM_I,
+                                    from_bool=True, to_bool=False, via_bool=False, normal_bool=False)
+            if len(temp_list) is not 0:
```
Comment (chiragjn) [inline models/crf/output_generation/city.py]: Consider using `if temp_list:`

File: models/crf/output_generation/city.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                list_json.append(temp_list)
+
+            else:
+                temp_list = check_label(reader_list=crf_data, reader_pointer=read_line,
+                                        begin_label=model_constant.CITY_TO_B, inner_label=model_constant.CITY_TO_I,
+                                        from_bool=False, to_bool=True, via_bool=False, normal_bool=False)
+                if len(temp_list) is not 0:
```
Comment (chiragjn) [inline models/crf/output_generation/city.py]: Consider using `if temp_list:`

File: models/crf/output_generation/city.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                    list_json.append(temp_list)
+
+                else:
+                    temp_list = check_label(reader_list=crf_data, reader_pointer=read_line,
+                                            begin_label=model_constant.CITY_VIA_B,
+                                            inner_label=model_constant.CITY_VIA_I, from_bool=False, to_bool=False,
+                                            via_bool=True, normal_bool=False)
+                    if len(temp_list) is not 0:
```
Comment (chiragjn) [inline models/crf/output_generation/city.py]: Consider using `if temp_list:`

File: models/crf/output_generation/city.py [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+                        list_json.append(temp_list)
+
+                    else:
+                        temp_list = check_label(reader_list=crf_data, reader_pointer=read_line,
+                                                begin_label=model_constant.CITY_NORMAL_B,
+                                                inner_label=model_constant.CITY_NORMAL_I, from_bool=False,
+                                                to_bool=False, via_bool=False, normal_bool=True)
+
+                        if len(temp_list) is not 0:
```
Comment (chiragjn) [inline models/crf/output_generation/city.py]: Consider using `if temp_list:`

File: models/models.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+class Models(object):
+    """""
+    This class will decide which model to choose based on its model_type
+    """""
+    def __init__(self):
+        """
+        Initalises the object
+        """
+
```
Comment (chiragjn) [inline models/models.py]: Consider ending empty constructor with `pass` for better readability

File: models/constant.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
+CRF_MODEL_TYPE = 'crf'
```
Comment (krupalmodi18) [inline models/constant.py]: Put comments throughout the file about what each and every constant means.  No one gets an idea about what city_o means

File: ner_v1/chatbot/entity_detection.py [hunk 7]
```diff
@@ -360,7 +360,7 @@ def get_city(message, entity_name, structured_value, fallback_value, bot_message
             return output_entity_dict_value(entity_value=structured_value, original_text=structured_value,
```
Comment (krupalmodi18) [inline ner_v1/chatbot/entity_detection.py]: update docstring

**Pull request merged**